src/lib/ThumbnailsSidebar.js

```diff
@@ -59,18 +60,20 @@ class ThumbnailsSidebar {
         this.renderNextThumbnailImage = this.renderNextThumbnailImage.bind(this);
         this.requestThumbnailImage = this.requestThumbnailImage.bind(this);
         this.thumbnailClickHandler = this.thumbnailClickHandler.bind(this);
+        this.onKeydown = this.onKeydown.bind(this);
```

Comment:
I think L62 needs specificity because there are potentially multiple things you could click in this sidebar. 

`onKeydown` is used as the name of the keydown handler in other places in Preview, from viewers to the find bar.